workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLauncherWindow.cs

```diff
@@ -0,0 +1,159 @@
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class MobileLauncherWindow : EditorWindow
+    {
+        private const string MobileSectionLabel = "Mobile Settings";
+        private const string iOSSectionLabel = "iOS";
+        private const string AndroidSectionLabel = "Android";
+        private const string DevelopmentTeamIdLabel = "Development Team Id";
+        private const string RuntimeIpLabel = "Local runtime IP";
+        private const string ConnectLocallyLabel = "Should connect locally?";
+
+        private MobileLaunchConfig launchConfig;
+
+        private string[] simulatorNames;
+        private string[] deviceNames;
+        
+        private int simulatorNameIndex;
+        private int deviceNameIndex;
+
+        [MenuItem("SpatialOS/Mobile launcher", false, 70)]
+        public static void ShowWindow()
+        {
+            var inspectorWindowType = typeof(EditorWindow).Assembly.GetType("UnityEditor.InspectorWindow");
+            var deploymentWindow = GetWindow<MobileLauncherWindow>(inspectorWindowType);
+            deploymentWindow.titleContent.text = "Mobile Launcher";
+            deploymentWindow.titleContent.tooltip = "A tab for managing your mobile client-workers.";
+            deploymentWindow.Show();
+        }
+
+        private void OnEnable()
+        {
+            launchConfig = new MobileLaunchConfig();
+            deviceNames = launchConfig.availableDevices.Keys.ToArray();
+            simulatorNames = launchConfig.availableSimulators.Keys.ToArray();
+        }
+        
+        public void OnGUI()
+        {
+            GUILayout.Label(MobileSectionLabel, EditorStyles.boldLabel);
+            using (new EditorGUI.IndentLevelScope())
+            {
+                EditorPrefs.SetString(
+                    launchConfig.RuntimeIpEditorPrefKey,
+                    EditorGUILayout.TextField(RuntimeIpLabel, launchConfig.RuntimeIp));
+                launchConfig.shouldConnectLocally = EditorGUILayout.Toggle(ConnectLocallyLabel, launchConfig.shouldConnectLocally);
+            }
+            
+            DrawHorizontalLine(10, new Color(0.7f, 0.7f, 0.7f, 1));
+            GUILayout.Label(AndroidSectionLabel, EditorStyles.boldLabel);
+
+            if (GUILayout.Button("Launch Android app"))
+            {
+                AndroidLaunchUtils.Launch(launchConfig.shouldConnectLocally);
+            }
+
+#if UNITY_EDITOR_OSX
```

Comment:
is there a way to check if the iOS package is installed? (or would there be more noticeable errors before we get here?)

Reply:
^same for Android

Reply:
if the iOS package is not installed, they won't be able to build their mobile workers -> any button they click here will fail immediately telling them to build their workers